gradle/libs.versions.toml

```diff
@@ -0,0 +1,88 @@
+[versions]
+aboutlibraries = "10.6.0"
+# @keep compose-compiler is not referenced by any dependency so needs to be kept explicitly
+compose-compiler = "1.4.2"
+compose-destinations = "1.7.22-beta"
+hilt = "2.45"
+junit-jupiter = "5.9.2"
+kotlin = "1.8.10"
+mockk = "1.13.4"
+sentry = "6.4.2"
+
+[libraries]
+aboutlibraries-compose = { module = "com.mikepenz:aboutlibraries-compose", version.ref = "aboutlibraries" }
+aboutlibraries-core = { module = "com.mikepenz:aboutlibraries-core", version.ref = "aboutlibraries" }
+androidx-activity-compose = "androidx.activity:activity-compose:1.6.1"
+androidx-constraintlayout-compose = "androidx.constraintlayout:constraintlayout-compose:1.0.1"
+androidx-core-ktx = "androidx.core:core-ktx:1.9.0"
+androidx-core-splashscreen = "androidx.core:core-splashscreen:1.0.0"
+androidx-hilt-navigation-compose = "androidx.hilt:hilt-navigation-compose:1.0.0"
+androidx-lifecycle-runtime-ktx = "androidx.lifecycle:lifecycle-runtime-ktx:2.5.1"
+androidx-navigation-compose = "androidx.navigation:navigation-compose:2.5.3"
+androidx-security-crypto-ktx = "androidx.security:security-crypto-ktx:1.1.0-alpha03"
+androidx-test-core = "androidx.test:core:1.5.0-rc01"
+androidx-test-espresso-core = "androidx.test.espresso:espresso-core:3.5.1"
+androidx-test-ext-junit = "androidx.test.ext:junit:1.1.5"
+compose-bom = "androidx.compose:compose-bom:2023.01.00"
```

Comment:
Compose bill of materials is now the recommended way to use the latest stable versions of compose dependencies since they will be versioned independently, see https://developer.android.com/jetpack/compose/bom/bom